package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@mhdawson I just guessed on the values for this.  We can make it whatever.  I left some of the fields blank too.